tools/python/boutiques/schema/descriptor.schema.json

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
how about using `git mv` to preserve the history of the file? (ignore this comment if you did it, github seems to show it as a new file...)